### ISSUE TITLE
Is it dead?

### ISSUE THREAD

**Issue by marwin1991** [davidB/yuicompressor-maven-plugin#99]
Is this project dead? Any alternatives?

**Comment by davidB** [comment]
Yes, no one seems interested to help/contribute to the project. And, IMHO, the base project com.yahoo.platform.yui::yuicompressor is far to be up-to-date with current js practice.

Sorry. TBH I didn't though this plugin was always in use.

**Comment by vicmosin** [comment]
Is there any working alternative which is up-to-date?

**Comment by melloware** [comment]
Yes we have a plugin that is up to date: https://github.com/primefaces-extensions/resources-optimizer-maven-plugin

Docs:  https://github.com/primefaces-extensions/primefaces-extensions.github.com/wiki/Maven-plugin-for-web-resource-optimization
